tests/integration/tests/test_sanity_telemetry.py

```diff
@@ -0,0 +1,123 @@
+"""
+This test suite goal is to validate end-to-end cloud security telemetry payloads.
+The suite assumes that the environment is already deployed and all integrations,
+such as KSPM (Kubernetes Security Posture Management) and CSPM (Cloud Security Posture Management),
+have been created.
+The goal of this suite is to perform basic sanity checks to verify the telemetry fetchers are working
+ as expected.
+"""
+# pylint: disable=redefined-outer-name
```

Comment:
When testing flake8 has part of the pre-commit test, the use of "redefined-outer-name" can collide with the usage of a pytest fixture.
https://stackoverflow.com/a/54045715

Reply:
@CohenIdo, To address this, you just need to name the fixture. For example, you can define the fixture like this:
```
@pytest.fixture(scope="module", name="telemetry_data")
def cloud_security_telemetry_data():
    """Fixture to fetch telemetry data"""
    telemetry_payload = get_telemetry(elasticsearch)
    telemetry_object = munchify(telemetry_payload[0])
    return telemetry_object.stats.stack_stats.kibana.plugins.cloud_security_posture
```

Then, use the named fixture in your test function:
```
@pytest.mark.sanity
def test_telemetry_indices(telemetry_data):


@pytest.mark.sanity
def test_telemetry_cloud_account_stats(telemetry_data):

```